app/Http/Controllers/Adm/Mship/Account.php

```diff
@@ -325,6 +326,17 @@ public function postBanAdd(CreateRequest $request, AccountData $account)
             ->withSuccess('You have successfully banned this member.');
     }
 
+    public function getBans(AccountData $account)
+    {
+        $bans = BanData::orderBy('created_at', 'DESC')
+            ->where('type', '80') //Only Show Local Bans
```

Comment:
Should use BanData::TYPE_LOCAL (just in case we change it or something) instead of the 80. If your feeling up to it, you could even implement this as a scope in the Account\Ban model (:)))

Reply:
@atoff The fact that this scope was already there and I didn't use it... makes me sad... Sorted!